src/labeler.ts

```diff
@@ -22,69 +23,69 @@ export async function run() {
     const syncLabels = !!core.getInput('sync-labels');
     const dot = core.getBooleanInput('dot');
 
-    const prNumber = getPrNumber();
-    if (!prNumber) {
+    const prNumbers = getPrNumbers();
+    if (!prNumbers) {
       core.info('Could not get pull request number from context, exiting');
```

Comment:
Now we may receive `prNumber` not only from context. 
```suggestion
      core.info('Could not get pull request number, exiting');
```